Project_2/djangoProject/cars/files/cars/api_detail.py

```diff
@@ -0,0 +1,33 @@
+import requests
+account = {
+    "email": "test.test@victorysquarepartners.com",
+    "password": "Y4WUzqYFu3Dif8K",
```

Comment:
We should avoid having credentials hardcoded. A better approach would be to do the following: 

1. Create a file called `.env` in the project directory (djangoProject probably) (don't forget about the dot in the file name)
2. Add the email and the password in the file (API_ACCOUNT_EMAIL=your.email@domain.com)
3. In the project settings.py file create two variables (API_ACCOUNT_EMAIL and API_ACCOUNT_PASSWORD, for example)
4. Use `os.getenv` to obtain the values from the .env file
5. Whenever you need them in a project you can use them like `settings.API_ACCOUNT_PASSWORD`

If, for some reason, this does not work (probably due to Windows), at least save the email and password in a normal text file and DO NOT add the file to git. This would prevent unintended access of the API in case anyone sees this repo, or this repo is made public.

I would also recommend to add the API URLs in the `.env` file for the same reason.